ParquetClassLibrary/Sandbox/MapChunk.cs

```diff
@@ -457,12 +438,15 @@ public override string ToString()
             {
                 for (var y = 0; y < DimensionsInParquets.y; y++)
                 {
-                    representation.Append(
-                        _collectableLayer[x, y]?.ToString()
-                        ?? _furnishingLayer[x, y]?.ToString()
-                        ?? _blockLayer[x, y]?.ToString()
-                        ?? _floorLayer[x, y]?.ToString()
-                        ?? "@");
+                    representation.Append(ParquetID.None != _collectableLayer[x, y]
+                            ? _collectableLayer[x, y].ToString()
+                            : ParquetID.None != _furnishingLayer[x, y]
+                                ? _furnishingLayer[x, y].ToString()
+                                : ParquetID.None != _blockLayer[x, y]
+                                    ? _blockLayer[x, y].ToString()
+                                    : ParquetID.None != _floorLayer[x, y]
+                                        ? _floorLayer[x, y].ToString()
+                                        : "@");
```

Comment:
nit: @ is the adventurer symbol, an empty space is the nothing symbol... in Nethack. This is probably better than having empty, though, since empty is by definition not visible. Just airing a super silly thought. :V

Reply:
Thanks!  Fixed in 90b88db.